tests/test-sources/plugins/languages/treesitter/treesitter.nix

```diff
@@ -1,14 +1,66 @@
+{ pkgs, ... }:
 {
+  default = {
+    plugins.treesitter = {
+      enable = true;
+
+      settings = {
+        auto_install = false;
+        ensure_installed = [ ];
```

Comment:
Good spot. I assumed the default was `"all"` but this [seems](https://github.com/nvim-treesitter/nvim-treesitter/blob/64f6f0ab4e3f613aa682eb5fe29c5025db500ddd/lua/nvim-treesitter/configs.lua#L23) [correct](https://github.com/nvim-treesitter/nvim-treesitter/blob/64f6f0ab4e3f613aa682eb5fe29c5025db500ddd/lua/nvim-treesitter/configs.lua#L609-L614).